build/KoreBuild.ps1

```diff
@@ -33,6 +35,14 @@ if ($env:KOREBUILD_DOTNET_VERSION)
 {
     $dotnetVersion = $env:KOREBUILD_DOTNET_VERSION
 }
+if ($env:KOREBUILD_DOTNET_SHARED_RUNTIME_CHANNEL)
+{
+    $sharedRuntimeChannel = $env:KOREBUILD_DOTNET_SHARED_RUNTIME_CHANNEL
+}
+if ($env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION)
+{
+    $sharedRuntimeVersion = $env:KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION
```

Comment:
This PR is so that we can use the environment variables to pass in version of the shared runtime when mirroring packages: https://github.com/aspnet/CoreCLR/commit/6f978e00c069add0c72bf9ead4bb44bbe08c95f5#diff-9a1b942859508c1b036a14226c0d9803R94